src/multi_layer_chunker.py

```
@@ -330,190 +320,191 @@ def _create_layer2_sections(self, extracted_doc) -> List[Chunk]:
-            chunk_contexts = self.context_generator.generate_contexts_batch(chunks_to_contextualize)
+            if not section_chunks:
+                continue
 
-            # Create Chunk objects with contexts
-            for (raw_chunk, metadata), context_result in zip(
-                chunks_to_contextualize, chunk_contexts
-            ):
-                section = metadata["section"]
-                idx = metadata["idx"]
+            # Create Chunk objects for each split
+            for chunk_idx, chunk_text in enumerate(section_chunks):
```

Ruff (0.14.3)

Loop control variable `chunk_idx` not used within loop body

Rename unused `chunk_idx` to `_chunk_idx`

(B007)